examples/README.md

```diff
@@ -271,6 +263,21 @@ The following platforms are currently supported:
 
 To add a new platform definition, add a new class to `krun/platform.py`.
 
+## Developer Mode
+
+If you are working in Krun itself, you may find the `--develop` switch useful:
+
```

Comment:
This is a bit unclear. How about something like: 

```
If you are making changes to Krun itself (for example, to add a new platform or virtual machine definition), you may find the `--develop` switch useful. 
This will cause Krun to run with the following modifications:
```